cli/app/internal/runtimeattach/activity.go

```
@@ -41,5 +44,18 @@ func SubscribeActivities(ctx context.Context, req ActivityRequest) (Activities,
+	if !req.AttentionNotificationsSupported {
+		return Activities{Session: sessionSub, Prompt: promptSub}, nil
+	}
+	if req.Attention == nil {
+		_ = promptSub.Close()
+		_ = sessionSub.Close()
+		Release(req.Runtime, req.SessionID, req.OwnerID)
+		return Activities{}, errors.New("attention notification service is required")
+	}
+	attentionSub, err := req.Attention.SubscribeSessionAttentionNotifications(ctx, serverapi.AttentionSessionNotificationSubscribeRequest{SessionID: req.SessionID, IncludePendingPromptSnapshot: true})
+	if err != nil {
+		return Activities{Session: sessionSub, Prompt: promptSub}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 56) but it returns nil

(nilerr)